src/components/characters/characterList.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { fetchCharacters } from '../services/fetch-characters';
+
+export default class Character extends Component {
+  state = {
+    characters: []
+  }
+
+  componentDidMount = async() => {
+    const characters = await fetchCharacters();
+
+    this.setState({ characters });
+  }
+
+  render() {
+    const thisCharacter = this.state.characters.map(character => (
+      <li key={character._id}>
+        <a href={`/${character._id}`}>
```

Comment:
You should use the <Link> component built into react-router-dom instead of <a> tags